src/__test__/manifest.loader.test.ts

```diff
@@ -0,0 +1,108 @@
+import o from 'ospec';
+import { Manifest } from '../manifest';
+import { isManifestsDifferent, ManifestLoader } from '../manifest.loader';
+
+o.spec('ManifestLoader', () => {
+  // Common test data (some manifests are used in different tests)
+
+  const manifestA: Manifest = JSON.parse(
+    `{
```

Comment:
Why are you using `JSON.parse` here?

Reply:
What should I use instead to deserialize the JSON to the Manifest object?

Reply:
JSON is javascript, so you can just paste it into the code without the quotes.